pkg/diagnostics/grpc_tracing.go

```diff
@@ -248,14 +267,10 @@ func SpanContextToGRPCMetadata(ctx context.Context, spanContext trace.SpanContex
 	return grpcMetadata.AppendToOutgoingContext(ctx, GRPCTraceContextKey, string(traceContextBinary))
 }
 
-func isInternalCalls(method string) bool {
-	return strings.HasPrefix(method, "/dapr.proto.internals.")
-}
-
 // spanAttributesMapFromGRPC builds the span trace attributes map for gRPC calls based on given parameters as per open-telemetry specs.
-func spanAttributesMapFromGRPC(appID string, req interface{}, rpcMethod string) map[string]string {
+func spanAttributesMapFromGRPC(appID string, req any, rpcMethod string) map[string]string {
 	// RPC Span Attribute reference https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/rpc.md
-	m := map[string]string{}
+	m := make(map[string]string, 8)
```

Comment:
I think you picked 8 because it looks like there are 8-ish unique keys below; can we add a note on why the map is deliberately sized? 😄